packages/api-openfin/src/window.js

```diff
@@ -52,19 +52,25 @@ const optionsMap = {
   'y': 'defaultTop'
 };
 
+const windowStates = {
+  MAXIMIZED: 'maximized',
+  MINIMIZED: 'minimized',
+  RESTORED: 'restored'
+};
+
 const checkWindowOpen = (win, windowMethod, reject) => {
   if (win) {
-    windowMethod();
+    return windowMethod();
   } else {
-    reject();
+    return reject(new Error('window does not exist'));
```

Comment:
The error in electron is `The window does not exist or the window has been closed`. Think they should both be the same, either way is fine